plugins/sy-personio-jobs-source-plugin/index.js

```diff
@@ -0,0 +1 @@
+// noop
```

Comment:
that's common for local gatsby plugins, otherwise it's not recognized as such (or did that change 😬)